lib/auth.js

```diff
@@ -110,9 +110,9 @@ class AuthenticatedRequest {
    *
    * @returns {Promise<string>} Promise of a valid authentication header
    */
-  async getTSV1Header(urlString, options, userID) {
```

Comment:
is `objectID` pattern used elsewhere? If so might be worth it to keep this as `userID` given it will cause less mental strain when switching/developing on Tozny go  and JS repos.

Reply:
this was me basically switching from go to javascript; the convention in our JS repos is `objectId` so I moved back towards consistency here

Reply:
+1 for userID

Reply:
hmm, I think it flies in the face of `clientId` and `apiKeyId` within this repo.